wicket-util/src/main/java/org/apache/wicket/util/encoding/UrlDecoder.java

```diff
@@ -16,25 +16,24 @@
  */
 package org.apache.wicket.util.encoding;
 
+import java.io.ByteArrayOutputStream;
 import java.io.UnsupportedEncodingException;
 import java.nio.charset.Charset;
 import java.nio.charset.IllegalCharsetNameException;
 import java.nio.charset.UnsupportedCharsetException;
 
 import org.apache.wicket.util.lang.Args;
-import org.apache.wicket.util.string.Strings;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * Adapted from java.net.URLDecoder, but defines instances for query string decoding versus URL path
+ * Adapted from Spring Framework's UriUtils class, but defines instances for query string decoding versus URL path
  * component decoding.
  * <p/>
  * The difference is important because a space is encoded as a + in a query string, but this is a
  * valid value in a path component (and is therefore not decode back to a space).
- * 
- * @author Doug Donohoe
- * @see java.net.URLDecoder
+ *
+ * @author Thomas Heigl

```

Comment:
I happened to be looking at Wicket source code again and noticed this change.  I'm the original author of the UrlDecode, originating back in 2008 in https://issues.apache.org/jira/browse/WICKET-1627.

This @author change is not cool.  It should at least be:

```
@author Doug Donohoe
@author Thomas Heigl
```

CC: @theigl